pkg/openthread/contrib/openthread.c

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2017 Fundaci�n Inria Chile
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @author  José Ignacio Alamos <jialamos@uc.cl>
```

Comment:
encoding issue in this file ? I also saw that in another one above. Please check file encoding (everywhere), or use ascii characters only)